index.js

```diff
@@ -36,7 +58,20 @@ class TrezorKeyring extends EventEmitter {
     this.unlockedAccount = 0;
     this.paths = {};
     this.deserialize(opts);
-    TrezorConnect.manifest(TREZOR_CONNECT_MANIFEST);
+    
+    TrezorConnect.on('DEVICE_EVENT', (event) => {
+      if (event && event.payload && event.payload.features) {
+        this.model = event.payload.features.model;
+      }
+    });
+    TrezorConnect.init({ manifest: TREZOR_CONNECT_MANIFEST })
```

Comment:
@danjm Brilliant move with `DEVICE_EVENT` event listening!  Unfortunately the event was never firing for me.  What does work, however, is this change I've made, which puts the event listener first and then triggers the event via `init`.

Reply:
****